nextflow_schema.json

```diff
@@ -76,28 +76,24 @@
             "properties": {
                 "dehosting_idx": {
                     "type": "string",
-                    "default": "./databases/PhiPacHum_m2.idx",
```

Comment:
Can you also try adding `"exists": true` for the database paths, as in https://nextflow-io.github.io/nf-validation/nextflow_schema/nextflow_schema_specification/#exists

This does say it's ignored for S3 paths unless you set a parameter, but I'm not sure about Azure paths. It's something we can test out.